suisa_sendemeldung/settings.py

```diff
@@ -0,0 +1,222 @@
+"""Settings for SUISA Sendemeldung."""
+
+from __future__ import annotations
+
+from datetime import datetime, timedelta
+from enum import StrEnum
+from zoneinfo import ZoneInfo
+
+import typed_settings as ts
+
+_EMAIL_TEMPLATE = """
+Hallo SUISA
+
+Im Anhang finden Sie die Sendemeldung von $station_name für den $month $year.
+
+Diese Sendemeldung erfolgt gemäss den Bestimmungen "Gemeinsamer Tarif S 2026 - 2028" [1] unter
+Einhaltung der Einreichefrist für den Monat $month gemäss Ziffer 45 des Tarifs.
+
+In der Sendungsmeldung enthalten sind die unter Buchstaben G (Verzeichnisse) des Tarifs
+genannten Programmangaben, in elektronischer Form. Die Angaben richten sich nach dem
+standardisierten Format gemäss Anhang 1 "Vorlage Sendelisten Radio" des obengenannten Tarifs
+sowie der Abstimmung zwischen SUISA und $station_name.
+
+
+Diese Sendemeldung enthält unter anderem die unter Ziffer 34 geforderten Angaben:
+
+- Titel des Musikwerks
+- Name des Komponisten
+- Name des/der Hauptinterpreten bzw. Gruppenname oder Name des Orchesters (falls zutreffend)
+- Sendedatum
+- Sendedauer
+- Sendezeit
+- vom Sender der Aufnahme selbst zugewiesene Identifikationsnummer
+- Kennzeichnung der Eigenaufnahmen
+- ISRC der benützten Aufnahme (sofern zusammen mit der Aufnahme vom Lieferanten der Aufnahme irgendeiner Form mitgeteilt bzw. mitgeliefert)
+- Label
+
+Die folgenden subsidiären Angaben melden wir in allen Fällen wo diese vom Lieferanten der Aufnahme mitgeteilt bzw. mitgeliefert wurden, insbesondere in Fällen wo der Lieferant der Aufnahme uns keine ISRC mitgeteilt bzw. mitgeliefert hat.
```

Comment:
```suggestion
Die folgenden subsidiären Angaben werden in allen Fällen gemeldet, in denen diese vom Lieferanten der Aufnahme mitgeteilt oder mitgeliefert wurden, insbesondere in Fällen, in denen der Lieferant der Aufnahme keine ISRC mitgeteilt oder mitgeliefert hat.
```

Using passive voice as within the rest of the mail text.

Reply:
The passus "mitgeteilt bzw. mitgeliefert" was taken straight from the tarif, but changing it in our mail makes sense.